features/shipping/managing_shipping_methods/adding_shipping_method.feature

```diff
@@ -46,3 +46,18 @@ Feature: Adding a new shipping method
         And I add it
         Then I should be notified that it has been successfully created
         And the shipment method "FedEx Carrier" should appear in the registry
+
+    @ui @javascript
+    Scenario: Adding a new shipping method for channel
+        Given I want to create a new shipping method
+        When I specify its code as "FED_EX_CARRIER"
+        And I name it "FedEx Carrier" in "English (United States)"
+        And I describe it as "FedEx Carrier shipping method for United States" in "English (United States)"
+        And I define it for the "United States" zone
+        And I make it available in channel "United States"
+        And I choose "Flat rate per unit" calculator
+        And I specify its amount as 20
+        And I add it
+        Then I should be notified that it has been successfully created
+        And the shipment method "FedEx Carrier" should appear in the registry
```

Comment:
Well, I think it would be no problem to adjust it after that PR merge, I don't see any reason to couple those two PR's.